Demo/src/main/java/com/paypal/android/api/services/AuthApi.kt

```diff
@@ -0,0 +1,42 @@
+package com.paypal.android.api.services
+
+import com.paypal.android.BuildConfig
+import com.paypal.android.api.model.AuthToken
+import com.paypal.android.api.model.AuthTokenResponse
+import okhttp3.Credentials
+import retrofit2.http.Field
+import retrofit2.http.FormUrlEncoded
+import retrofit2.http.Header
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+interface AuthApi {
+
+    /**
+     * Creates a new OAuth Token that can be used for subsequent API requests.
+     *
+     * @see payPalAuthorization
+     */
+    @Headers("Accept: application/json", "Accept-Language: en_US")
+    @FormUrlEncoded
+    @POST("/v1/oauth2/token")
+    suspend fun postOAuthToken(
```

Comment:
Is this OAuth call for making the capture/authorize call after the PayPal module returns a result?

Reply:
@tdchow no. When I implemented this, i coulndt use the sample server, because the `clientId` on the server didnt have the correct scopes, so it would work on native. So i generate my own token now, to be able to create an order.
Basically is:  (create token oauth) -> (create order v2) -> (call checkout with orderid)

Reply:
Got it! This may not be needed anymore since the merchant sample server should be working now that the clientID and secret match. Would you be able to try it out and see if we can remove these calls?